CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# 0.2.0

```

Comment:
I think you can collapse all this into `0.1.0` since we haven't released any versions yet.

Reply:
Done.  I wasn't sure if intermediate commits were being published to a pre-release repo somewhere so thought I needed to update the version as I went.  I'll note the behavior for future.

Reply:
No automatic publishing/deploying. There's a "Deploy" button in each build job that lets you deploy to GitHub and the PowerShell Gallery.